cli.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python3
+import click
+import click_completion
+from pyintelowl.pyintelowl import IntelOwl
+from pyintelowl.cli import groups, cmds
+from pyintelowl.cli._utils import get_logger, ClickContext, get_netrc_obj
+
+
+# Enable auto completion
+click_completion.init()
+
+
+@click.group(context_settings=dict(help_option_names=["-h", "--help"]))
+@click.option("-d", "--debug", is_flag=True, help="Set log level to DEBUG")
+@click.pass_context
+def cli(ctx: ClickContext, debug: bool):
+    netrc, host = get_netrc_obj()
```

Comment:
if the `.netrc` file is not available, this would break

```
  File "/home/xx/pyintelowl/pyintelowl/cli/_utils.py", line 74, in get_netrc_obj
    netrc = Netrc()
  File "/home/xx/pyintelowl_client/lib/python3.6/site-packages/tinynetrc-1.3.0-py3.6.egg/tinynetrc.py", line 39, in __init__
    self._netrc = netrc.netrc(file)
  File "/usr/lib/python3.6/netrc.py", line 32, in __init__
    with open(file) as fp:
FileNotFoundError: [Errno 2] No such file or directory: '/home/xx/.netrc'

```

The hint is a good idea but it is placed after this error

Reply:
we could just do something like `import pathlib; pathlib.Path(".netrc").touch()` to create the file if it doesn't exist. Thoughts?